samples/client/petstore/scala-sttp/src/main/scala/org/openapitools/client/core/ApiInvoker.scala

```diff
@@ -0,0 +1,72 @@
+/**
+ * OpenAPI Petstore
+ * This is a sample server Petstore server. For this sample, you can use the api key `special-key` to test the authorization filters.
+ *
+ * The version of the OpenAPI document: 1.0.0
+ * 
+ *
+ * NOTE: This class is auto generated by OpenAPI Generator (https://openapi-generator.tech).
+ * https://openapi-generator.tech
+ * Do not edit the class manually.
+ */
+package org.openapitools.client.core
+
+import org.joda.time.DateTime
+import org.joda.time.format.ISODateTimeFormat
+import org.json4s.JsonAST.JString
+import org.json4s._
+import sttp.client._
+import org.openapitools.client.api.EnumsSerializers
+import org.openapitools.client.core.ApiInvoker.DateTimeSerializer
+import sttp.client.json4s.SttpJson4sApi
+import sttp.client.monad.MonadError
+
+class SttpSerializer(implicit val format: Formats = DefaultFormats ++ EnumsSerializers.all + DateTimeSerializer,
+                     implicit val serialization: org.json4s.Serialization = org.json4s.jackson.Serialization) extends SttpJson4sApi
+
+class HttpException(val statusCode: Int, val statusText: String, val message: String) extends Exception(s"[$statusCode] $statusText: $message")
```

Comment:
can we use sttp type / alias for status code instead?

Reply:
done